doc/parameters.md

```diff
@@ -194,18 +194,16 @@ It can be used to first load policies from a development directory or to load ex
 
 ### `APICAST_ENVIRONMENT`
 
-**Default**:
-**Value:**: string\[:<string>\]
-**Example**: production:cloud-hosted
+**Value:** string\[:<string>\]  
```

Comment:
I see that you removed the default value. I think that even if it's nil we should document it.

Reply:
@davidor I can revert this change, and we can address it separately. Sounds OK?

Reply:
:+1: